### PR TITLE
added config for bounding

### PR DESCRIPTION
**Pull request by gabrieloks**
This PR is related to the implementation of activation functions in the model (https://github.com/ecmwf/anemoi-models/pull/14). Here, I have just added the config entries needed for the usage of activation functions, but I have commented them so that they remain dormant. They can not be used until the feature is implemented in anemoi models.

### CODE REVIEW

**Comment by FussyDuck** [comment]
[CLA assistant check](https://bol-claassistant.ecmwf.int:443/ecmwf/anemoi-training?pullRequest=10) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://bol-claassistant.ecmwf.int:443/ecmwf/anemoi-training?pullRequest=10) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: gabrieloks<br/>:x: gmertes<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bol-claassistant.ecmwf.int:443/check/ecmwf/anemoi-training?pullRequest=10) it.</sub>

**Comment by mchantry** [comment]
No changelog update?

File: src/anemoi/training/config/data/zarr.yaml
Comment (HCookie) [inline src/anemoi/training/config/data/zarr.yaml:1]: Are these configuration changes directly related to the bounding?

File: src/anemoi/training/config/model/gnn.yaml
Comment (HCookie) [inline src/anemoi/training/config/model/gnn.yaml:1]: Should all configurations be using these bounding configs?
As in:
- Should we expect all basic models to be using `tp` & `cp`
- Do these boundings generalise well?

File: src/anemoi/training/config/data/zarr.yaml
Comment (mchantry) [reply src/anemoi/training/config/data/zarr.yaml:1]: Yes

File: src/anemoi/training/config/model/gnn.yaml
Comment (mchantry) [reply src/anemoi/training/config/model/gnn.yaml:1]: Generalise in what sense?

File: src/anemoi/training/config/data/zarr.yaml [hunk 1]
```diff
@@ -30,6 +30,11 @@ remapped:
 
 normalizer:
   default: "mean-std"
+  remap:
+    cp: tp
```
Comment (mchantry) [inline src/anemoi/training/config/data/zarr.yaml:34]: Can you explain this block?

File: src/anemoi/training/config/model/gnn.yaml
Comment (HCookie) [reply src/anemoi/training/config/model/gnn.yaml:1]: To other use cases, datasets, etc

Comment (mchantry) [reply src/anemoi/training/config/model/gnn.yaml:1]: CP has been commented out, as it's not generic. I think having a precipitation is generic, so would propose to leave it in.

**Review by mchantry — APPROVED** [review]
LGTM thanks for the hard work.

**Pull request merged**